script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -294,47 +352,54 @@ His one and only flaw is his love of women.
 [sha:7681a7dac155c16dd2a8f721c5bfd49f646998bf]{
 -- Page 809, Offset 13043.
 -- その孫……あくまで表向きはだが……であるマーリオゥは、特にラウレンティス枢機卿から覚えがよく、遅咲きの後継者だと言われている。
-Mario... who is known to the public as his grandson, is said to be a late-blooming successor. He is held in high regard, particularly by Cardinal Laurentis.
+// Reading 遅咲き is late_comer_ (young out of his brothers) rather than late _bloomer_ (he seems to have bloomed very early)
+Cardinal Laurentis holds Mario――who is ostensibly his grandson in name only――in high regard,
+ and there are whispers that he's likely to come from behind to claim the title of successor.
 }
 [sha:beecb60f3a4432254c78b4b903bcfe187b543397]{
 -- Page 810, Offset 13044.
 -- ベスティーノという姓はラウレンティス枢機卿がまだ片田舎に住んでいた頃の旧姓である。20人を超える養子……いや、兄たちを差し置いて、この姓を与えられた少年がどれほど期待されているかなど、信徒であれば説明されるまでもない。
-The surname Bestino is the old family name of Cardinal Laurentis, back from where he was still living in the remote countryside. All the faithful know very well just how much is expected of the young boy who was granted this surname, over more than twenty other adopted sons... no, over more than twenty of his brothers.
+The surname Bestino is Cardinal Laurentis's old family name, a memento from when he was still living in the backcountry.
+ It's obvious to any member of the clergy just how high Laurentis's expectations must be for this boy,
```

Comment:
drop comma even though it will result in a MEGA SENTENCE
_maybe_ comma after "surname".